test/functional/page_objects/visual_builder_page.ts

```diff
@@ -614,6 +614,13 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
       );
       return await comboBox.isOptionSelected(groupBy, value);
     }
+
+    public async setMetricsDataTimerangeMode(value: string) {
+      const timerangeMode = await find.byCssSelector(
```

Comment:
nit: please set  data-test-subj value for that component to use` testSubjects.find` instead of `byCssSelector`

Reply:
done